tests/network/probes.go

```diff
@@ -74,14 +74,14 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			return vmiReady(readVmi) == corev1.ConditionTrue
 		}
 
-		table.DescribeTable("should succeed", func(readinessProbe *v1.Probe, IPFamily corev1.IPFamily, isExecProbe bool, disableEnableCycle bool) {
+		table.DescribeTable("should succeed", func(readinessProbe *v1.Probe, ipFamily corev1.IPFamily, isExecProbe bool, disableEnableCycle bool) {
```

Comment:
Commit: tests/network/probes: use lowercase for local variable

I do not think there is such a convention needed. Local variables of a function cannot be exported anyway.
The convention I know of is to use capital letters for acronyms.
Anyway, I do not really mind, feel free to keep it.

Reply:
I explained why I prefer lowecase, but if anyone wants me to drop the commit, I'll do so.